apps/etterlatte-fordeler/src/main/kotlin/pdltjenester/PdlTjenesterKlient.kt

```diff
@@ -43,18 +44,16 @@ class PdlTjenesterKlient(private val client: HttpClient, private val apiUrl: Str
                         }
                     val feilFraPdl =
                         try {
-                            response.body<PdlFeil>()
+                            val feil = response.body<ExceptionResponse>()
+                            enumValueOf<PdlFeilAarsak>(feil.code!!)
                         } catch (e: Exception) {
                             throw samleExceptions(it.exceptions + e)
                         }
-                    when (feilFraPdl.aarsak) {
+                    when (feilFraPdl) {
                         PdlFeilAarsak.FANT_IKKE_PERSON ->
-                            throw PersonFinnesIkkeException(hentPersonRequest.foedselsnummer)
+                            throw FantIkkePersonException()
```

Comment:
Fint å ikkje ha fødselsnummer i loggane, og vi har vel kanskje ikkje noko anna nyttig å logge her heller. Får håpe det ikkje skjer så ofte da.

Reply:
Ja, jeg tenkte litt fram og tilbake her. Er åpen for begge deler, lekte litt med en nullable folkeregisteridentifikator som constructor-argument til feilen som ga en mer detaljert tekst med den maskerte identifikatoren. Kan endre til det hvis du mener det er bedre!